### DISCUSSION TITLE
can't make "catch all route $.tsx" working for / (root)

### DISCUSSION THREAD

**Discussion by GuyoST** [remix-run/remix#5362] [Q&A]
Hi every one, 

I would like any page on my website to show the same tsx file.
for example : 

domain.ext/page
domain.ext/toto
domain.ext/plouf/tata
domain.ext
domain.ext/meuh1/meuh2/meuh3

any page should reach the same component.

So I create a single file : `/routes/$.tsx` and it's working well except for `domain.ext`.

I realized that `domain.ext` is working only if I have a `/routes/index.tsx`.

So I created it, but of course `$.tsx` and `index.tsx` are identical.
Is there a way to have only one file that can catch any page even index ?

Thank you for your help 😁

**Comment by GuyoST** [comment]
Hello every one again, sorry to ask again, but I still haven't succeed about this...
What I would like is that any url of my website uses the same routes, and I didn't understand how to organize my routes to achieve this.
Can someone help me about it ?

Thank you :)

**Reply by salos1982** [reply]
I have the same issue

**Comment by sergiodxa** [comment]
`routes/$.tsx` will handle `/*` that means it handle any path after the `/`, but not `/` itself, that's why you also need `routes/_index.tsx`. This is how catch-all routes work in Remix and React Router.

If those routes are identical, you could re-export everything from one on the another, like doing:

```ts
export * from "~/routes/$";
export { default } from "~/routes/$";
```

Or if you want the index to be the source of truth:

```ts
export * from "~/routes/_index";
export { default } from "~/routes/_index";
```
